src/test/java/integration/EntriesApiTest.java

```diff
@@ -311,6 +315,25 @@ void getDynamicFieldsEntry_ReturnDynamicFields() {
     }
 
     @Test
+    void getEntryByFullPath_ReturnRootFolder() {
+        FindEntryResult entry = repositoryApiClient.getEntriesClient().getEntryByPath(repoId, rootPath, false).join();
+        assertNotNull(entry);
+        assertTrue(entry.getEntry().getId() == 1);
```

Comment:
Please try to not use assertTrue/assertFalse, when other more specific assertions can be used.
Here, if the assertion fails, the test report just says that this equality does not hold. It doesn't say what the entry's Id was. So, does not help that much.
If it is replaced with the following, it's better and more helpful if it fails.
`assertEquals(1, entry.getEntry().getId());`